### ISSUE TITLE
Example of markdown.mustache does not contain field names for data types

### ISSUE THREAD

**Issue by alexei-osipov** [kongchen/swagger-maven-example#7]
Table template for section `Data Types` in `templates\markdown.mustache` does not contain contain column with information about field (property) name.

Current code (table headers):
```html
<tr>
  <th>type</th>
  <th>required</th>
  <th>access</th>
  <th>description</th>
  <th>notes</th>
</tr>
```

**Comment by alexei-osipov** [comment]
I think it's supposed to have an extra column column like
```html
        <th>property</th>
```
```html
        <td>{{name}}</td>
```

**Comment by jacakk** [comment]
Yes, but I think that plugin is not generating proper table with "property" column. Am I wrong?

**Comment by alexei-osipov** [comment]
Plugin not generates "property" column because it just messed in template. And this can be fixed with just editing template file.

**Comment by sdhanavath** [comment]
I have just started using the swagger-maven-example to use the maven-plugin for generating swagger.json and static html and not successful in doing that, I observed that few files are missing as per the web.xml configuration. Please refer attached image for the error.

Can you please help me on this.

Thanks in advance!

![swagger-maven-example_missing_code](https://cloud.githubusercontent.com/assets/2561364/8375512/629c0fd8-1c1b-11e5-9108-177de7ab782e.png)
